website/content/docs/nia/release-notes/0-5-0.mdx

```diff
@@ -0,0 +1,356 @@
+- `source_input`: v0.7
+- `source_includes_var`: v0.7
+
+The documentation below will refer to the deprecated configurations by their new name.
+
+### Address `services` field and `service` block
+
+The `services` field and its associated `service` block are frequently an exception to task configuration use-cases and add complexity and restrictions.
+
+The `services` field is the only task condition that is not configured through the `condition` block. It is also the only additional module input that is not configured through a `module_input` block. This leads to a lot of inconsistent relationships with other `condition` and `module_input` blocks.
+
+We deprecated the `services` field and its associated `service` filter to remove these exceptions and complicated relationship with other `condition` and `module_input` blocks:
+
+| Deprecated Config | Replacement Config | Description |
+| --- | --- | --- |
+| `services` field | `condition "services"` or `module_input "services"` block | The services to provide module input and occasionally act as the task condition |
+| `service` block | fields moved into the `condition "services"` and `module_input "services"` block | The configuration block to provide additional filtering on the services in the `services` field |
+
+The `services` field and its associated `service` block will be removed in a future major release after v0.7.
+
+## Further Details on Deprecations
+
+### Deprecate `source` field
+
+The `source` field in the task configuration is deprecated and can be directly renamed to `module`.
+
+Example:
+
+<CodeTabs heading="Deprecate `source` field" tabs={["Deprecated", "New"]}>
+
+<CodeBlockConfig highlight="4">
+
+```hcl
+task {
+  name     = "services_task"
+  services = ["api", "web"]
+  source   = "path/to/module"
+}
+```
+
+</CodeBlockConfig>
+
+<CodeBlockConfig highlight="4">
+
+```hcl
+task {
+  name     = "services_task"
+  services = ["api", "web"]
+  module   = "path/to/module"
+}
+```
+
+</CodeBlockConfig>
+
+</CodeTabs>
+
+### Deprecate `source_input` block
+
+The `source_input` block in the task configuration is deprecated and can be directly renamed to `module_input`.
+
+Example:
+<CodeTabs heading="Deprecate `source_input` block" tabs={["Deprecated", "New"]}>
+
+<CodeBlockConfig highlight="7">
+
+```hcl
+task {
+  name   = "scheduled_task"
+  module = "path/to/module"
+  condition "schedule" {
+    cron = "* * * * Mon"
+  }
+  source_input "consul-kv" {
+    path = "my_key"
+  }
+}
+```
+
+</CodeBlockConfig>
+
+<CodeBlockConfig highlight="7">
+
+```hcl
+task {
+  name   = "scheduled_task"
+  module = "path/to/module"
+  condition "schedule" {
+    cron = "* * * * Mon"
+  }
+  module_input "consul-kv" {
+    path = "my_key"
+  }
+}
+```
+
+</CodeBlockConfig>
+
+</CodeTabs>
+
+
+### Deprecate `source_includes_var` field
+
+The `condition` block's `source_includes_var` field is deprecated for all types of conditions and can be directly renamed to `use_as_module_input`.
```

Comment:
```suggestion
The `condition` block's source_includes_var field is deprecated for all types of conditions and can be directly renamed to `use_as_module_input`.
```

Reply:
Wanted to double-check removing the backticks on here? The "source_includes_var" here refers to the configuration. It's placed in text rather than the header CodeTabs header. Let me know if I'm missing something

Reply:
@lornasong  just in the headings for the codetabs `heading="blah blah `source.....`"

Reply:
Yes, this suggestion is actually not in the header of CodeTabs but is in a text block. It looks like it might be unintentional and wanted to double-check.